### PR TITLE
Add searchable attributes for query tool

### PR DESCRIPTION
**Pull request by loicgasser**

### CODE REVIEW

File: chsdi/locale/it/LC_MESSAGES/chsdi.po [hunk 1]
```diff
@@ -2372,9 +2372,33 @@ msgstr "Linea di volo"
 msgid "ch.swisstopo.lubis-luftbilder-dritte-firmen"
 msgstr "Immagini aeree privati"
 
+msgid "ch.swisstopo.lubis-luftbilder-dritte-firmen.bgdi_flugjahr"
+msgstr "Année de vol "
```
Comment (procrastinatio) [inline chsdi/locale/it/LC_MESSAGES/chsdi.po]: c'est du Français!

File: chsdi/locale/it/LC_MESSAGES/chsdi.po [hunk 2]
```diff
@@ -2372,9 +2372,33 @@ msgstr "Linea di volo"
...
+msgstr "Numero immagine"
+
+msgid "ch.swisstopo.lubis-luftbilder-dritte-firmen.ort"
+msgstr "Luogo"
+
 msgid "ch.swisstopo.lubis-luftbilder-dritte-kantone"
 msgstr "Immagini aeree dei cantoni"
 
+msgid "ch.swisstopo.lubis-luftbilder-dritte-kantone.bgdi_flugjahr"
+msgstr "Année de vol "
```
Comment (procrastinatio) [inline chsdi/locale/it/LC_MESSAGES/chsdi.po]: Italiano, per piacere!

File: chsdi/locale/it/LC_MESSAGES/chsdi.po [hunk 3]
```diff
@@ -2396,6 +2420,18 @@ msgstr "Località"
 msgid "ch.swisstopo.lubis-luftbilder_schwarzweiss"
 msgstr "Immagini aeree swisstopo b/n"
 
+msgid "ch.swisstopo.lubis-luftbilder_schwarzweiss.bgdi_flugjahr"
```
Comment (procrastinatio) [inline chsdi/locale/it/LC_MESSAGES/chsdi.po]: Ancora..

File: chsdi/locale/en/LC_MESSAGES/chsdi.po [hunk 4]
```diff
@@ -2373,9 +2373,33 @@ msgstr "Flying line"
 msgid "ch.swisstopo.lubis-luftbilder-dritte-firmen"
 msgstr "Aerial images privates"
 
+msgid "ch.swisstopo.lubis-luftbilder-dritte-firmen.bgdi_flugjahr"
+msgstr "Flugjahr"
```
Comment (procrastinatio) [inline chsdi/locale/en/LC_MESSAGES/chsdi.po]: English

**Comment by procrastinatio** [comment]
Technically looks good, but some italian and english translation are missing (noted a few inline)

**Comment by loicgasser** [comment]
Thanks for noticing. I ll fix it.

**Comment by loicgasser** [comment]
@pfanguin Could I ask you to replace those tranlsations

en:
ch.swisstopo.lubis-luftbilder-dritte-firmen.bgdi_flugjahr
ch.swisstopo.lubis-luftbilder-dritte-kantone.bgdi_flugjahr
ch.swisstopo.lubis-luftbilder_schwarzweiss.bgdi_flugjahr

-> Flight year

fr:
ch.swisstopo.lubis-luftbilder-dritte-firmen.bgdi_flugjahr
ch.swisstopo.lubis-luftbilder-dritte-kantone.bgdi_flugjahr
ch.swisstopo.lubis-luftbilder_schwarzweiss.bgdi_flugjahr

-> Anno volo

Thanks! Let me know when it's ok, so that I can update this PR.

**Comment by pfanguin** [comment]
Done !

**Comment by loicgasser** [comment]
Thanks!

**Comment by loicgasser** [comment]
for en I only have Flight :x instead of Flight year.

**Comment by pfanguin** [comment]
Sorry, normaly it's good

**Comment by loicgasser** [comment]
ok comments addressed

**Pull request merged**